index.html

```diff
@@ -3,11 +3,187 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Capstone</title>
     <link rel="stylesheet" href="style.css">
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
 </head>
 <body>
-    <h1>hello</h1>
-    <h2>hoo</h2>
+    <!-- NAV -->
+    <header>
+        <nav class="top-nav">
+            <a>Pages</a>
+            <a>Logout</a>
+        </nav>
+        <nav class="main-nav">
+            <img src="./images/img/mBlack.png"/>
+            <i class='bx bx-menu' id="menu-icon"></i>
+            <div class="nav-links hidden">
+                <a href="./about.html">About</a>
+                <a>Program</a>
+                <a>Join</a>
+                <a>Faqs</a>
+                <a>News</a>
+                <a>Login</a>
+            </div>
```

Comment:
Kindly fix the following error 👇


https://github.com/ibtisam-adil/Capstone/assets/118625824/22a9281a-ff56-4603-9a2e-b05d7f7c21ed